src/styles/main.scss

```
@@ -1 +1,71 @@
-// put code here
+$clock-size: 80vh;
+$clock-center-size: 50px;
+$hour-turn: 3;
+$minute-turn: 0;
+$time-animation: 4s;
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: #b4eeaf;
+}
+
+.clock {
+  width: $clock-size;
+  height: $clock-size;
+  border: 5px dotted #000;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__center {
+    position: absolute;
+    width: $clock-center-size;
+    height: $clock-center-size;
+    border-radius: 50%;
+    background-color: #efc405;
+    z-index: 3;
+  }
+
+  &__hour {
+    position: absolute;
+    width: 20px;
+    height: $clock-size / 4;
+    background-color: #1b7ad4;
+    transform: rotate($hour-turn * 30deg);
+    animation: rotate-arrow $time-animation * 12 linear infinite;
+    z-index: 1;
+  }
+
+  &__minute {
+    position: absolute;
+    width: 10px;
+    height: $clock-size / 2;
+    background-color: #3dd41b;
+    transform: rotate($minute-turn * 6deg);
+    animation: rotate-arrow $time-animation linear infinite;
+    z-index: 2;
+  }
+}
+
+@keyframes rotate-arrow {
+  0% {
+    transform:
+      rotate(0deg)
+      translateY(-50%);
+  }
+
+  100% {
+    transform:
+      rotate(360deg)
+      translateY(-50%);
+  }
+}
```

```suggestion
@keyframes rotate-arrow {
  0% {
    transform:
      rotate(0deg)
  }

  100% {
    transform:
      rotate(360deg)
  }
}
```
move translateY(-50%); directly to class its not needed in animation if you dont change it